plugins/TPCSDAction.cpp

```diff
@@ -55,9 +54,23 @@ namespace sim {
     G4ThreeVector CumulativeMeanMomentum{};
     G4int CumulativeNumSteps{};
 
-    G4Step const* previousStep{};
     std::map<int, G4double> padRowCentralRadii{};
 
+    struct stepInfo {
+      bool isValid{};
```

Comment:
```suggestion
      bool isValid{false};
```
I would assume that should be the default (and also is IIRC), but this way people would not have to guess and we are not at the mercy of the compiler never changing.